framework/src/materials/ParsedMaterialHelper.C

```diff
@@ -104,6 +129,55 @@ ParsedMaterialHelper<is_ad>::functionParse(
     const std::vector<PostprocessorName> & postprocessor_names,
     const std::vector<std::string> & tol_names,
     const std::vector<Real> & tol_values)
+{
+  const std::vector<MooseFunctorName> empty_functor_vector;
+  const std::vector<std::string> empty_string_vector;
+  functionParse(function_expression,
+                constant_names,
+                constant_expressions,
+                mat_prop_expressions,
+                postprocessor_names,
+                tol_names,
+                tol_values,
+                empty_functor_vector,
+                empty_string_vector);
+}
+
+template <bool is_ad>
+void
+ParsedMaterialHelper<is_ad>::functionParse(const std::string & function_expression,
```

Comment:
It doesn't look like this overload is used. I would vote to just not add it if the more encompassing overload below is all we need

Reply:
I removed the overload.